build-system/tasks/extension-generator/bento/amp-__component_name_hyphenated__/amp-__component_name_hyphenated__.md

```diff
@@ -114,6 +122,10 @@ Description of action. Use cases of `action-name`. Include all the nuances, such
 
 Description of event. Use cases of event-name. Include all the nuances, such as: `amp-__component_name_hyphenated__` needs to be identified with an `id` to work.
 
+#### Valid AMP
+
+Syntax and argument details for use in fully valid AMP pages.
```

Comment:
The valid AMP snippet in this section should include a `layout` attribute and/or `width` and `height` attributes. The Bento snippet should exclude them.